styles/amplenote/catppuccin.user.css

```diff
@@ -0,0 +1,234 @@
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain("amplenote.com") {
+  body:not(.theme-light) {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+
+  body.theme-light {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[@@lookup][@rosewater];
+    @flamingo: @catppuccin[@@lookup][@flamingo];
+    @pink: @catppuccin[@@lookup][@pink];
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    &.themeable {
+      --color-background-primary: @base;
+      --color-background-secondary: @mantle;
+      --color-background-tertiary: @mantle;
+
+      --color-background-popup: @base;
+      --color-background-dialog: @mantle;
+      --color-background-header: @mantle;
+      --color-background-nav: @mantle;
+      --color-background-nav-active: @surface0;
+      --color-background-search-bar: @crust;
+      --color-background-nav-menu: @mantle;
+      --color-background-nav-menu-divider: @crust;
+      --color-background-nav-avatar: @crust;
+      --color-background-popup-small: @surface0;
+      --color-background-reversed: @surface0;
+
+      --color-background-element-high-contrast: fade(@overlay2, 40%);
+      --color-background-element-medium-contrast: fade(@accent-color, 40%);
+      --color-background-element-low-contrast: @crust;
+
+      --color-background-note-menu-button: @surface0;
+      --color-background-banner: @surface0;
```

Comment:
They also have a `--color-background-banner-alt` variable, which is used under the `.account-status-banner` element in this case:

![CleanShot 2024-07-08 at 19 33 37@2x](https://github.com/catppuccin/userstyles/assets/47499684/b6bc33e4-d5a6-4cab-9489-1a669b9076a9)